pkg/apis/sippy/v1/types.go

```diff
@@ -69,11 +70,30 @@ type FailureGroup struct {
 	TestFailures int    `json:"testFailures"`
 }
 
-type Release struct {
-	Release              string
-	Status               string
-	GADate               *time.Time
-	DevelopmentStartDate *time.Time
+type Release struct { // this is the Release that gets cached
+	Release                 string
+	Status                  string
+	GADate                  *time.Time
+	DevelopmentStartDate    *time.Time
+	PreviousRelease         string
+	InstallIndicatorsSchema int64
+	ReleaseBools
+}
+type ReleaseBools struct { // boolean modifiers for a release
+	// ExcludeComponentReadiness excludes the release as an option for component readiness comparisons.
+	ExcludeComponentReadiness bool `json:"exclude_component_readiness"`
+	// ExcludeSippyClassic excludes the release in the Sippy Classic UI.
+	ExcludeSippyClassic bool `json:"exclude_sippy_classic"`
+	// ExcludeFetchData skips syncing job data from the release to sippy's DB
+	ExcludeFetchData bool `json:"exclude_fetch_data"`
+	// ExcludeMetrics skips metrics collection and analysis for the release.
+	ExcludeMetrics bool `json:"exclude_metrics"`
+	// IncludePullRequests enables the Sippy Classic pull request UI for this release
+	IncludePullRequests bool `json:"include_pull_requests"`
+	// ExcludeFeatureGates omits sippy classic link for seeing release feature gates
+	ExcludeFeatureGates bool `json:"exclude_feature_gates"`
+	// ExcludePayloadTags omits sippy classic link for seeing release tags for payloads
+	ExcludePayloadTags bool `json:"exclude_payload_tags"`
 }
```

Comment:
I am not a huge fan of booleans like this. The mixed Exclude and Include logic is also hard to follow. If you're going to use booleans, then I think they should all go one direction (i.e. Include or Exclude), but I'd rather not have any.  

Bigquery can do arrays (I think it calls them repeated fields), could we just have a single column that's a list of a release's capabilities?